docs/broker/README.md

```diff
@@ -68,7 +68,7 @@ For development purposes or to use the latest from the repository, use [`ko`](ht
 ko apply -f config/broker/
 ```
 
-Follow the [Broker-Trigger](../../samples/broker-trigger) to deploy a basic example of a topology.
+Follow one of the [Broker-Trigger](../../samples/broker-trigger) examples to deploy a basic topology.
```

Comment:
I'd suggest linking this straight to the `quick-setup`. The other samples are for more customized setups